Classes/Models/Basic Models/Generated/CPProtoPodfile.m

```diff
@@ -28,17 +21,29 @@ - (void)setWorkspace:(NSString*)workspace; {
   [self setBackingStoreValue:workspace forKey:@"workspace"];
 }
 
-- (BOOL)generateBridgeSupport; {
-  id value = [self backingStoreValueForKey:@"generate_bridge_support"];
-  return [value boolValue];
+- (NSArray*)sources; {
+  id value = [self backingStoreValueForKey:@"sources"];
+  return CPCoerceToArray(value);
 }
 
-- (void)setGenerateBridgeSupport:(BOOL)generateBridgeSupport; {
-  [self setBackingStoreValue:[NSNumber numberWithBool:generateBridgeSupport] forKey:@"generate_bridge_support"];
+- (void)setSources:(NSArray*)sources; {
+  [self setBackingStoreValue:sources forKey:@"sources"];
+}
+
+- (NSArray*)plugins; {
+  id value = [self backingStoreValueForKey:@"plugins"];
+  return CPCoerceToArray(value);
+}
+
+- (void)setPlugins:(NSArray*)plugins; {
+  [self setBackingStoreValue:plugins forKey:@"plugins"];
 }
 
 - (BOOL)setArcCompatibilityFlag; {
   id value = [self backingStoreValueForKey:@"set_arc_compatibility_flag"];
+  if (!value) {
+    return YES;
+  }
```

Comment:
Agreed. Could either override in the concrete class or add some logic to the rake file to be able to add default values when these are generated.